src/content/roadmap/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Ethereum roadmap
+description: The path to more scalability, security and sustainability for Ethereum.
+lang: en
+template: roadmap
+image: ../../assets/upgrades/merge.png
+alt: "Ethereum roadmap"
+summaryPoints:
+  - Learn about how Ethereum will evolve into the future
+  - Read why Ethereum is still being worked on
+  - Understand how Ethereum is setting up for long term success
+buttons:
+  - label: Further updates
+    toId: what-changes-are-coming
+  - label: Past updates
+    to: /history/
+    variant: outline
+---
+
+Ethereum is a powerful platform for global coordination, but it is not finished! An ambitious set of improvements will upgrade Ethereum from its current form into a fully scaled, maximally resilient platform. These upgrades are laid out in the Ethereum roadmap.
+
+**To learn about previous upgrades to Ethereum, such as EIP-1559 or The Merge, please visit our [History](/history/) page**
+
+## What changes are coming to Ethereum? {#what-changes-are-coming}
+
+The Ethereum roadmap outlines the specific improvements that will be made to protocol in the future. Overall, the roadmap will bring the following benefits to Ethereum users:
+
+<CardGrid>
```

Comment:
Overall, I think the content of these cards should be pithier (or just generally a little shorter).